ports/libsimpleservo/src/capi.rs

```diff
@@ -302,6 +303,14 @@ impl HostTrait for HostCallbacks {
         (self.0.on_title_changed)(title_ptr);
     }
 
+    fn on_allow_navigation(&self, url: String) -> bool {
+        debug!("on_allow_navigation");
+        let url = CString::new(url).expect("Can't create string");
+        let url_ptr = url.as_ptr();
+        mem::forget(url);
+        return (self.0.on_allow_navigation)(url_ptr);
```

Comment:
Remove the return to be consistent with the rest of the file.

Reply:
`return (self.0.on_allow_navigation)(url_ptr);` is equivalent to `(self.0.on_allow_navigation)(url_ptr)` (notice that the missing semicolon).

```rust
fn a() -> bool { return true; }
```
is the same as
```rust
fn a() -> bool { true }
```

Reply:
> (notice that the missing semicolon).

Oh. Yeah. I forgot the semicolon.
I have updated that.

Reply:
Actually, since we'll cannot make it consistent with others (others do have semicolon) I think `return` would provide better readability? WDYT?